python/py-hgexperimental/Portfile

```
@@ -30,6 +31,13 @@ if {${name} ne ${subport}} {
                         port:py27-cython \
                         port:lz4
 
+    pre-configure {
```

I would have said that `post-patch` makes more sense:
```suggestion
    post-patch {
```
however ...